src/lib/utils/normalizeText.js

```diff
@@ -1,107 +1,22 @@
-//
-// Method to normalize size of fonts across devices
-//
-// Some code taken from https://jsfiddle.net/97ty7yjk/ &
-// https://stackoverflow.com/questions/34837342/font-size-on-iphone-6s-plus
-//
-// author: @xiaoneng
-// date: 14/10/2016
-// version: 03
-//
+import { Dimensions } from 'react-native'
 
-const React = require('react-native') // eslint-disable-line no-undef
-const { PixelRatio, Dimensions } = React
+const { height, width } = Dimensions.get('window')
 
-const pixelRatio = PixelRatio.get()
-const deviceHeight = Dimensions.get('window').height
-const deviceWidth = Dimensions.get('window').width
+const DESIGN_WIDTH = 360
+const DESIGN_HEIGHT = 640 - 24
 
-// -- Testing Only --
-// const fontScale = PixelRatio.getFontScale();
-// const layoutSize = PixelRatio.getPixelSizeForLayoutSize(14);
-// console.log('normalizeText getPR ->', pixelRatio);
-// console.log('normalizeText getFS ->', fontScale);
-// console.log('normalizeText getDH ->', deviceHeight);
-// console.log('normalizeText getDW ->', deviceWidth);
-// console.log('normalizeText getPSFLS ->', layoutSize);
+const windowHeight = height > DESIGN_HEIGHT ? DESIGN_HEIGHT : height
+const windowWidth = width > DESIGN_WIDTH ? DESIGN_WIDTH : width
```

Comment:
on desktop we limit the height and width of viewport, so we need to use those in the calculations

Reply:
@sirpy I tried to apply this code:
![code](https://user-images.githubusercontent.com/49862004/66309440-4b171d80-e912-11e9-9623-764ce0cbc648.png)

and it seems like looks good: but one small this is not ok:
<img width="479" alt="1" src="https://user-images.githubusercontent.com/49862004/66309514-831e6080-e912-11e9-9b24-ef64a1e9728f.png">

Reply:
this doesnt look like the correct code and font sizes

Reply:
@sirpy I found the limitations for app width and height. it is - height: 844, width: 475. So I used those. Isn't that correct?

Reply:
1. you are missing the desktop condition
2. design width+height are not the max height/width

Reply:
windowHeight = isDesktopOrTablet && height>maxHeight ? maxHeight : height

Reply:
@sirpy I tried to apply condition from your last comment. It's looking good on mobile (simulator) but not on desktop/tablet (https://prnt.sc/pjfzsg). There is the same issue that I already described earlier in this conversation. Also, I tried to apply this new formula for normalizeText function to other branches where was some change with styles for feed cards and I have unsatisfied results too. So, I still think that we should return the previous formula and we can avoid these troubles. And if it really needed to change the formula - then it may be done as a separate task.